recipes/propka/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "propka" %}
+{% set version = "3.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a3/7b/e935ce37cbf9ac8fe23ec05f8072fb3a63582b20fb01ef0c24502b5d10ba/PROPKA-3.1.0.tar.gz
+  md5: e2c2397165c874cabd0196188319baac
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - propka31 -h
+
+about:
+    home: https://github.com/jensengroup/propka-3.1
+    license: LGPL
+    license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Please read the documentation about this section.

LICENSE can be taken out of the upstream tarball.

Please also use `conda skeleton pypi propka` to generate those recipes. You will see you get a much better one.